gpt_researcher.py

```
@@ -0,0 +1,701 @@
+                        ''')]
+    else:
+        messages = [SystemMessage(content=sys_content),HumanMessage(content=f'''{usr_prompt}\n
+                Given Title: {title}\n
+                Given Abstract: {abstract}
+            ''')]
+    chat = ChatOpenAI(model="gpt-3.5-turbo")
+
+
+    return chat.batch([messages])[0].content
+
+from pydantic import BaseModel, Field
```

Ruff (0.8.2)

Redefinition of unused `BaseModel` from line 25

Remove definition: `BaseModel`

(F811)